config/_extras.yml

```diff
@@ -18,6 +18,12 @@ themesAd:
   - name: 'Pixel Lite - Bootstrap 4 UI Kit'
     img: /assets/img/themes/pixel-lite-free-ui-kit-themesberg.jpg
     url: 'https://themesberg.com/product/ui-kits/pixel-lite-free-bootstrap-4-ui-kit'
+  - name: 'AdminKit Pro - Web UI Kit & Dashboard Template'
```

Comment:
This can't, other affiliates will ask why they have to give us a cut of each sale.